src/components/SearchForm.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div class="wrapper">
+    <p>
+      <span class="text_netflix">netflix</span>
+      <span class="text_routlette">roulette</span>
+    </p>
+    <h2>{{ headerText }}</h2>
+    <form class="search-form" @submit="handleSubmit" :style="style">
+      <div class="search-wrapper">
+        <input
+          class="search-form__input"
+          type="text"
+          name="searchValue"
+          placeholder="Search..."
+          v-model="searchValue"
+        />
+        <input type="submit" value="Search" class="search-form__submit" />
+      </div>
+      <SwitcherComponent
+        v-on:switcher-data="handleSwitcher"
+        text="SEARCH BY"
+        name="searchBy"
+        :checkedValue="searchType"
```

Comment:
just value?

Reply:
v-model=\"searchType"

and inside SwitcherComponent instead of checkedValue -> value, switcherData -> onChange
or redefine model for component
model: {
    prop: 'checkedValue',
    event: 'switcherData'
  },
https://v2.vuejs.org/v2/api/#model